src/components/CurriculumList.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <section class="screen-box">
+    <Header
+      title="カリキュラム一覧"
+      url="/"
+    />
+    <main class="screen-box__main curriculum-list">
+      <div
+        v-for="(curriculum, index) in curriculumList"
+        :key="index"
+        :style="styleObject( curriculum )"
+        class="curriculum-box"
+      >
+        <p>{{ curriculum.title }}</p>
+        <div
+          class="curriculum-box__subcontent"
+          @click="$router.push(`/curriculum/${index}`)"
```

Comment:
This should be changed to named route after being merged with curriculum detail.